edge-modules/edgehub-proxy/Cargo.toml

```diff
@@ -7,17 +7,18 @@ edition = "2018"
 [dependencies]
 clap = "2.31"
 env_logger = "0.5"
-failure = "0.1"
-failure_derive = "0.1"
-hyper = "0.12.36"
+hyper = "0.14"
+thiserror = "1"
 log = "0.4"
 serde_json = "1.0"
-tokio = "0.1"
+tokio = { version = "1", features = ["rt"] }
 url = "1.7"
+chrono = "0.4"
 
-edgelet-http = { path = "../../edgelet/edgelet-http" }
 edgelet-utils = { path = "../../edgelet/edgelet-utils" }
-workload = { path = "../../edgelet/workload" }
+edgelet-http-workload = { path = "../../edgelet/edgelet-http-workload" }
```

Comment:
From a quick glance at the changes, I don't see anything from this crate being used. Could you double-check that it's necessary?

Reply:
Good catch. That and http-common are leftovers from when I was trying to take a different path to replace the workload api code. Removed both